synapse/storage/databases/main/search.py

```diff
@@ -55,7 +55,7 @@ def store_search_entries_txn(self, txn, entries):
                     entry.event_id,
                     entry.room_id,
                     entry.key,
-                    entry.value,
+                    entry.value.replace("\u0000", "\u0020"),
```

Comment:
I think this is just clearer as a space, what do you think?

```suggestion
                    entry.value.replace("\u0000", " "),
```

Reply:
Yes I think that makes sense!

Reply:
Yeah I probably would have used `\x00` personally, but they're all equivalent.